pkg/authorization/workspace_content_authorizer_test.go

```diff
@@ -103,23 +103,23 @@ func TestWorkspaceContentAuthorizer(t *testing.T) {
 			requestedWorkspace: "root:unknown",
 			requestingUser:     newUser("user-1"),
 			wantDecision:       authorizer.DecisionDeny,
-			wantReason:         "clusterworkspace not found",
+			wantReason:         `workspace "root"|"unknown" not found`,
```

Comment:
Do you want the quotes around path and name? Most/all of our other code like this does `%s|%s`

Reply:
ah no, that's an artifact of using `%q|%q`, fixed.